Gruntfile.js

```diff
@@ -142,7 +145,11 @@ module.exports = function(grunt) {
                     browserifyOptions: {
                         debug: false
                     },
-                    transform: ["babelify", ["loose-envify", { "NODE_ENV": "production" }]]
+                    transform: [
+                        "babelify",
+                        ["uglifyify",{global:true}],
```

Comment:
There is already an uglify task being run, so if you want to use this version, please remove the other task.